lib/semantics/expression.h

```diff
@@ -326,11 +326,12 @@ class ExpressionAnalyzer {
       const parser::Call &, bool isSubroutine);
   std::optional<characteristics::Procedure> CheckCall(
       parser::CharBlock, const ProcedureDesignator &, ActualArguments &);
-  const Symbol *ResolveGeneric(
-      const Symbol &, ActualArguments &, const semantics::Scope &);
+  const Symbol *ResolveGeneric(const Symbol &, ActualArguments &);
```

Comment:
```ResolveGeneric()``` only seems to be used in expression.cc.  Should it be in expression.h?

Reply:
It needs to be a member function so that `Say` is available.